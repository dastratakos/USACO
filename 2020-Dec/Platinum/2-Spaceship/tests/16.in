60 59 60
110110111010111101010010111101101011110011100111111110111111
110111110111011101100110110010111101011010101111001010111111
001011110000001101110110001101110000110011111101111111100011
110011001011110011111011100011000110000011011100011010101100
011010111101100111101111111011011011110101111101000110100000
000110101111111101111110001011011011100101011010101111111111
111011011100101101111111111111110001011110111011111110110010
101011110010111111011110111100111000001011011111011101111000
011111111001001011101000110001111100101111111011101011111111
011010101001011101101110110100001011111111100111111110111111
111110010110110111111111000111011111110111101110001110111111
101101010101011111110011101100000011101111001110110111111110
101110101111111001111110111011110111110100101010010111101001
100111111100001111101100101011001111111111110111111111110011
101111110011011100111110110000110111111001011111110100101111
110111100011011110110100111111111111001010011111111001111100
110101101111111001101110101011100111111111001111110110100111
001101010110011000000110001011111000101110110111111101111010
111111100111111110111011010101011111100111011010010110001000
111111010101111110011100111110010011000101000101111111110111
010111001010111001110011011111010100111011001101111110101101
111111001001110101110110111111001011111011101111010011001110
100101001100111110111111101001101111110010110101110101110011
111001010111111111100110110111011011111011100111011011110111
100011110100111101110101011101010101111111110011111101101101
011001101100101111111001001110110101110111110111011101101100
011010111111011111111101101000111011010110110001111100111111
111100111110001100011111111111011110110111010111101110110111
111001001011101010100111011111011110110111110111111111011011
101100101001000001111111111111011111111101110111100010101001
111110011110111011111111110011111100011011100000111100010110
010100100000111000011111001111000111111000111100110101111111
010100101111110111111110101101101111000111111001011000110111
000010010010111110101100100110011111111001011001110100111000
101111001111111111011100111111110111011111110101011111110000
001110110110101000010110100111101111001001111111011111101110
100111110110111111001111001011111100110001011111100111101011
011111111000001001111110111000011000010110111000011101011101
110110101001101011110010001111100111001101101001011101001110
011110111110111111100101011011001010111111000011111011010101
110111111111101011110100010111100110111001111001000010011011
011111110101111111110100111011101100110110111011011111000101
110111100111111001101111101111100101111111111101110010010111
001111010101011010011110111101111111111010011111101101111000
111111011011011101101110110111011000011000111101101100000111
100010110111110111011011111011100100011111111111111110011110
010111010110011000100111111011110110011111110110001001011111
111101000000011100111111101110011110101011001110011001111011
111111110010111110100111011010111111111110111111110110011001
111111011111110101111101000110101100011100111111111110111011
011000100110010010111011010111000111011111101101111110010000
010111011001001111001000111101011010111111001011010110010101
111100111110110111110010111110101101111010001010011110101101
001011011111111110101101111110111011010101101010101111010111
110111100111011010111000100000011111111011011111111111111111
111001111011111011111111101011011110111110011101100011111101
000011111011101001010111100000111101111100110111010111101010
001100011010101011010010111110101000011010110111011010111110
101110111111001100110110100101111111010110011011101100110001
111001111110110100111111010110011011100010011111111110110011
3 8 4 28
2 36 4 26
41 55 4 6
53 14 5 25
53 53 2 8
2 5 3 16
15 3 3 5
5 34 5 57
4 45 4 47
1 57 1 15
3 12 3 28
29 25 13 34
55 7 41 14
2 46 10 10
5 41 5 59
3 49 3 48
1 13 5 32
2 31 7 15
26 28 25 28
4 3 5 41
32 31 4 14
5 51 33 17
5 49 3 29
58 6 2 47
4 18 3 52
2 8 5 11
6 55 3 48
5 36 1 39
5 42 2 25
3 54 44 50
56 51 1 25
2 26 4 54
2 42 4 45
3 18 5 4
1 9 2 55
1 57 4 1
5 17 2 42
4 53 38 30
4 58 4 31
2 34 2 39
5 26 3 37
2 56 5 58
2 27 2 17
3 2 4 35
1 54 2 37
1 53 1 37
3 2 2 55
4 25 1 51
1 52 2 10
3 59 5 45
1 14 5 1
37 34 2 51
5 42 3 10
3 42 25 56
4 8 2 43
52 45 5 39
1 13 4 9
3 30 2 38
5 2 5 22
5 57 5 57
