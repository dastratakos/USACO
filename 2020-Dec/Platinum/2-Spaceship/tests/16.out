119599851
253913577
192068128
987241667
924960313
436457742
724101316
549666696
342328117
825892071
193936249
136972566
722067170
145153605
369047574
357405181
442609305
903226205
815305478
121408453
811391654
417637386
513206169
530032797
267177991
279173611
421026697
528408526
87707564
638674058
431698467
233456238
28536898
316891870
434409754
930828239
524473237
667375761
174303659
592577015
749652148
858386038
746701818
312092021
356654453
286464435
953841235
415111946
760503180
335325205
666494435
932417089
743827564
577515121
426324211
652925274
977751842
921683968
512462297
991297146
